pytest_plugins/video_cleanup.py

```diff
@@ -22,7 +22,7 @@ def _clean_video(session_id, test):
 
         if settings.ui.grid_url and session_id:
             grid = urlparse(url=settings.ui.grid_url)
-            infra_grid = Host(hostname=grid.hostname, ipv6=settings.server.is_ipv6)
```

Comment:
Just curious to know, how we would connect to IPv6 host now, without ipv6 kwargs?

Reply:
Robottelo runs on OpenShift where we have both network stacks available, therefore, robottelo should be able to connect to it just fine. 

That reminds me that we should make tweaks to the robottelo config as well.